src/backup-all-indexes.sh

```diff
@@ -67,6 +95,42 @@ backup_index ()
   curl -w "\n" -sS -XDELETE ${INDEX_URL}
 }
 
+backup_index ()
+{
+  : ${1:?"Error: expected index name passed as parameter"}
+  local INDEX_NAME=$1
+  local SNAPSHOT_URL=${REPOSITORY_URL}/${INDEX_NAME}
+  local INDEX_URL=${DATABASE_URL}/${INDEX_NAME}
+
+  grep -q SUCCESS <(curl -sS ${SNAPSHOT_URL})
+
+  if [ $? -ne 0 ]; then
+    echo "$(now): Scheduling snapshot."
+    # If the snapshot exists but isn't in a success state, delete it so that we can try again.
+    grep -qE "FAILED|PARTIAL|IN_PROGRESS" <(curl -sS ${SNAPSHOT_URL}) && curl -sS -XDELETE ${SNAPSHOT_URL}
+    # Indexes have to be open for snapshots to work.
+    curl -sS -XPOST "${INDEX_URL}/_open"
+
+    curl --fail -w "\n" -sS -XPUT ${SNAPSHOT_URL} -d "{
+      \"indices\": \"${INDEX_NAME}\",
+      \"include_global_state\": false
+    }" || return 1
+
+    echo "$(now): Waiting for snapshot to finish..."
+    timeout "${WAIT_SECONDS}" bash -c "until grep -q SUCCESS <(curl -sS ${SNAPSHOT_URL}); do sleep 1; done" || return 1
+    #increase backup counter
+    let bk++
+  fi
+
+
+         echo "$(now): Ensuring ${index_name} is backuped..."
```

Comment:
Can you clean up the indentation and the newline noise here?

(Also, I'm a little unclear on what that message is even doing here)